sw/device/silicon_creator/manuf/lib/sram_start.S

```diff
@@ -186,7 +196,10 @@ sram_start:
   call test_main
 
   // Notify the host that we are done.
-  li  sp, SRAM_MAGIC_SP_EXECUTION_DONE
+  // li   sp, SRAM_MAGIC_SP_EXECUTION_DONE
```

Comment:
This commented line is no longer required, correct?  The value is instead loaded from the stack, and then moved into `sp` for examination by the jtag host?

Reply:
oops, meant to delete that. thanks. fixed.